test/e2e/framework/clusterresourcebinding.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2023 The Karmada Authors.
```

Comment:
2024

Reply:
How time flies~

Reply:
Why the CI doesn't find out the problem？

Reply:
CI can only find out the file which misses the license header

Reply:
I think we should fix this, but in another PR.